domain/src/main/kotlin/no/nav/su/se/bakover/domain/Sak.kt

```diff
@@ -450,6 +450,11 @@ class SakFactory(
     }
 }
 
+data class BegrensetSakerInfo(
```

Comment:
Kunne vi funnet et bedre navn på denne? 🤔 Litt vanskelig å skjønne hva den egentlig inneholder. Litt usikker på hvor `BegrensetSak`-termin i utganspunktet kom fra. Vi ønsker vel bare å hente info om en søknad for alder/uføre eksisterer? Kanskje vi kunne funnet på et navn som etterspeiler det litt bedre?

Reply:
Ja, jeg er ikke fornøyd med navnet her jeg heller. Så vidt jeg forstår hensikten med `BegrensetSakinfo` (the O.G.)  går dette på om det finnes en åpen sak med relevans for en eventuell ny søknad. Hva med noe som `ÅpneSakerInfo`? Det illustrerer ikke at det er en veldig begrenset informasjon man får om saken, men sier mer om hensikten bak dataene. Eller noe som `AlleredeÅpenSak(er)ForPerson`? Litt usikker på hva jeg tenker om flertall vs. entall egentlig, siden man får potensielt tilbake to saker, men de er adskilt på type